packages/fxa-payments-server/src/routes/Subscriptions/en.ftl

```diff
@@ -30,6 +32,7 @@ pay-update-manage-btn = Manage
 ## $date (Date) - The date for the next time a charge will occur.
 
 sub-next-bill = Next billed on { $date }
+sub-next-bill-2 = Your next bill of <amount></amount> is due <strong>{ $date }</strong>
```

Comment:
@bcolsson do you have an example of how this should be done?

At the moment I'm trying to implement it following [these guidelines](https://github.com/projectfluent/fluent.js/wiki/React-Overlays).

Reply:
@StaberindeZA: Thanks for taking care of the other issues. For this one, I'm just comparing against other strings in the file like `sub-item-cancel-sub`, so I could be misunderstanding things.

Right now with `<amount></amount>` and no text or placeholder inside of it, while the elements may receive html markup there won't be any text appearing within it.

Looking through whats in the code now it seems like you should assign in `vars` something like: `{{ amount: nextBillAmount }}`

Then for the the ftl text you could have:
`sub-next-bill-2 = Your next bill of <amount>{ $amount }</amount> is due <strong>{ $date }</strong>`

Please double check with someone on your team though if this is accurate.

Reply:
Hi @bcolsson 

In this case `amount` is not actually a variable it's a React.Element, returned by PriceDetails. I'm following the [Fluent.js Markup](https://github.com/projectfluent/fluent.js/wiki/Localized#markup) guidelines to implement this.

Please note that PriceDetails displays the price in many different ways, depending on the conditions. (With/out tax, with/out interval, etc.) Also just wanted to mention that PriceDetails has its own localization logic, so that won't need to be handled in the string we are discussing now.